htdocs/langs/fr_FR/users.lang

```diff
@@ -129,3 +129,6 @@ IPLastLogin=IP dernière connexion
 IPPreviousLogin=Connexion précédente IP
 ShowAllPerms=Afficher toutes les lignes d'autorisation
 HideAllPerms=Masquer toutes les lignes d'autorisation
+EventUserRibModifiedHTML=Vos coordonnées bancaires viennent d'être modifiées. Si vous n'êtes pas à l'origine de cette modification, veuillez rapidement en informer votre administrateur.
```

Comment:
add translations for en_US, they are used to create all other translations

Reply:
Done